source/administration/config-settings.rst

```diff
@@ -2263,6 +2263,21 @@ SAML
 .. note::
    In line with Microsoft ADFS guidance we recommend `configuring intranet forms-based authentication for devices that do not support WIA <https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/configure-intranet-forms-based-authentication-for-devices-that-do-not-support-wia>`_.
 
+Use New SAML Library
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**True**: Enable an updated SAML Library, which does not require the XML Security Library (xmlsec1) to be installed.
+
+**False**: Continue using the existing implementation which uses the XML Security Library (xmlsec1).
+
+.. code-block:: sh
+
+  "ExperimentalSettings": {
```

Comment:
This section may not be correctly formatted. See below "Enable Login With SAML" setting on how config settings are normally formatted.

Reply:
Thanks Amy, will do.

Reply:
I've changed it in line with the rest of the examples. @sbishel, can you confirm that the default is ``false`` with options ``true`` and ``false``?

Reply:
Just a reminder about this confirmation, @sbishel, when you have a moment. :)